src/main/resources/querydsl_checks_base.xml

```diff
@@ -90,6 +91,11 @@
   </module>
 
   <module name="SuppressWarningsFilter"/>
+  <module name="SuppressionCommentFilter">
+    <property name="offCommentFormat" value="CSOFF\: ([\w\|]+)"/>
+    <property name="onCommentFormat" value="CSON\: ([\w\|]+)"/>
```

Comment:
it already clashes with turning checkstyle off entirely: `<[[CHECKSTYLE:OFF]]>: ([\w]+) `.
Also, since it IS used to turn off a rule, it is descriptive enough.
And it won't be used that much, so verbosity I think is not that bad?